src/components/TranslationBannerLegal.tsx

```diff
@@ -38,54 +33,39 @@ const TranslationBannerLegal = ({
   }, [originalPagePath, shouldShow])
 
   return (
-    <Box
-      as="aside"
-      position="fixed"
-      display={isOpen ? "block" : "none"}
-      bottom={{ base: 0, md: 8 }}
-      insetInlineEnd={{ base: 0, md: 8 }}
-      zIndex="99"
+    <aside
+      className={cn(
+        "fixed z-50",
+        "bottom-0 md:bottom-8",
+        "right-0 md:right-8",
+        isOpen ? "block" : "hidden"
+      )}
     >
-      <Flex
-        bg="infoBanner"
-        color="black300"
-        justify="space-between"
-        maxW={{ base: "full", md: "600px" }}
-        maxH="full"
-        p={4}
-        borderRadius="sm"
-        boxShadow={{
-          base: `-4px 10px 0px ${text} 10%`,
-          md: "rgba(0, 0, 0, 0.16) 0px 2px 4px 0px",
-        }}
+      <div
+        className={cn(
+          "relative flex justify-between",
+          "w-full md:max-w-[600px]",
+          "rounded-sm p-4",
+          "bg-info-banner text-black-300",
+          "shadow-[rgba(0,0,0,0.16)_0px_2px_4px_0px] md:shadow-md"
+        )}
       >
-        <Flex direction="column" m={4} mt={{ base: 10, sm: 4 }}>
-          <Flex
-            align={{ base: "flex-start", sm: "center" }}
-            flexDirection={{ base: "column-reverse", sm: "row" }}
-            mb={4}
-          >
-            <Heading
-              as="h3"
-              fontSize={{ base: "1.25rem", md: "1.5rem" }}
-              fontWeight="bold"
-              lineHeight="100%"
-            >
+        <div className="m-4 mt-[40px] flex flex-col sm:mt-4">
+          <div className="mb-4 flex flex-col-reverse items-start sm:flex-row sm:items-center">
```

Comment:
lets use a gap in the parent to simplify and make the spacing more consistent
```suggestion
          <div className="flex flex-col-reverse items-start sm:flex-row sm:items-center">
```